src/text/index.tsx

```diff
@@ -47,19 +48,25 @@ const Text = ({
   children,
   display = TextDisplayType.BODY,
   tag = TextTagType.SPAN,
+  textColor,
   newlineToBr = true,
 }: TextProps) => {
   const baseClassName = 'kirk-text'
   const displayClassName = `${baseClassName}-${display}`
   const Tag = tag
 
+  const cssColorRegex = /^#([0-9a-f]{3}|[0-9a-f]{6})$/i
```

Comment:
Can you add a test with valid / invalid colors ?

Reply:
Done